packages/target-tauri/src-tauri/src/settings.rs

```diff
@@ -30,13 +35,14 @@ pub(crate) fn load_and_apply_desktop_settings_on_startup(app: &AppHandle) -> any
     apply_zoom_factor(app)?;
     // TODO: activate tray icon based on `minimizeToTray`
     apply_content_protection(app)?;
+    apply_autostart(&app)?;
```

Comment:
```suggestion
    if let Err(err) = apply_autostart(&app)
        .context("failed to apply autostart") {
        // Not too critical, let's just log.
        error!("{err}")
    };
```

Just a suggestion, I'm not sure if it's best to handle the error here.

This will also require importing `log::error`.